tools/releaseBuild/azureDevOps/templates/linux.yml

```diff
@@ -94,6 +131,24 @@ jobs:
         PackageVersion: $(Version)
         sourceScanPath: '$(PowerShellRoot)/tools'
 
+  - ${{ if eq(variables.build,'deb') }} :
+    - template: EsrpSign.yml@ComplianceRepo
+      parameters:
+        buildOutputPath: '$(System.ArtifactsDirectory)/pwshLinuxBuildMinSize'
+        signOutputPath: '$(System.ArtifactsDirectory)/pwshLinuxBuildMinSize'
+        certificateId: "CP-230012"
+        pattern: |
+          **\*.dll
```

Comment:
Do they request for signing DLLs too?

Reply:
All files that can be embedded signed are required to be signed